src/app/static/src/app/store/plottingSelections/plottingSelections.ts

```diff
@@ -22,16 +22,18 @@ export const initialBarchartSelections = {
     selectedFilterOptions: {}
 };
 
-export const initialPlottingSelectionsState = {
-  barchart: {...initialBarchartSelections}
+export const initialPlottingSelectionsState = (): PlottingSelectionsState => {
+    return {
+        barchart: {...initialBarchartSelections}
```

Comment:
I think technically we'd get away with this because the selectedFilterOptions are replaced rather than modified when they're initialised by the Barchart component. But might be safer to make initiaBarchartSelections a function too, same as initialSelectedChoroplethFilters.

Reply:
Thanks, agree good to be complete, it's future proofing if nothing else.